apps/cmp_mock_srv.c

```diff
@@ -0,0 +1,418 @@
+/*
+ * Copyright 2018-2020 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright Siemens AG 2018-2020
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or atf
+ * https://www.openssl.org/source/license.html
+ */
+
+#include "cmp_mock_srv.h"
+#ifdef OPENSSL_NO_CMP
+NON_EMPTY_TRANSLATION_UNIT
```

Comment:
This is the "old" way of doing things. It is better to handle this in the build.info file, i.e. if we are doing a "no-cmp" build then don't even bother trying to compile this file.

E.g. something like this:
````
IF[{- !$disabled{'cmp'} -}]
  $OPENSSLSRC=$OPENSSLSRC \
        cmp_mock_srv.c
ENDIF
````

Reply:
Thx for the good hint; done.